python/anvill/ida.py

```diff
@@ -605,6 +606,7 @@ class IDAProgram(Program):
   def __init__(self, *args):
     super(IDAProgram, self).__init__(_get_arch(), _get_os())
     self._functions = weakref.WeakValueDictionary()
+    self._dwarf = DWARFCore(idc.GetInputFilePath())
```

Comment:
Probably need to `try`...`except` in the event that this isn't an ELF file, support graceful degradation.